src/EPR.PRN.Backend.API/Services/PrnService.cs

```diff
@@ -94,6 +94,50 @@ public async Task UpdateStatus(Guid orgId, Guid userId, List<PrnUpdateStatusDto>
         await _repository.SaveTransaction(transaction);
     }
 
+    public async Task SavePrnDetails(SavePrnDetailsRequest prn)
+    {
+        try
+        {
+            Eprn prnEntity = new Eprn()
+            {
+                AccreditationNumber = prn.AccreditationNo!,
+                AccreditationYear = prn.AccreditationYear.ToString()!,
+                // CancelledDate = prn.CancelledDate, // This property /column does not exist on Eprn entity or DB table PRN
```

Comment:
commented code here

Reply:
yeah that is on purpose because in the mapping document there was no mapping for that property to DB table.